### PR TITLE
docs(tabs): decompose Static tabs demo

### PR DESCRIPTION
**Pull request by ghost**
docs(tabs): rename statis section to base section in tabs-demo
it's(Static tabs) the same as basic tabs 
related to #3267

### CODE REVIEW

**Review by EvilAlexei — CHANGES_REQUESTED** [review]
You should add a demo with `select` event :)

File: demo/src/app/components/+tabs/tabs-section.list.ts [hunk 1]
```diff
@@ -32,8 +32,8 @@ export const demoComponentContent: ContentSection[] = [
     outlet: ExamplesComponent,
     content: [
       {
-        title: 'Static tabs',
-        anchor: 'tabs-static',
+        title: 'Basic tabs',
```
Comment (EvilAlexei) [inline demo/src/app/components/+tabs/tabs-section.list.ts]: Please rename to just `Basic`

**Review by EvilAlexei — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
'You should add a demo with select event '
this demo was added to another pr (issue - #3664, pr - #3668)

**Review by EvilAlexei — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Comment by ghost** [comment]
I have sent this pr to the test, cause I merged conflicts

**Review by IlyaSurmay — APPROVED** [review]

**Review by valorkin — APPROVED** [review]

**Pull request merged**